modules/mono/build_scripts/build_assemblies.py

```diff
@@ -315,12 +324,7 @@ def main():
         sys.exit(1)
 
     exit_code = build_all(
-        msbuild_tool,
-        module_dir,
-        output_dir,
-        args.godot_platform,
-        args.dev_debug,
-        args.push_nupkgs_local,
+        msbuild_tool, module_dir, output_dir, args.godot_platform, args.dev_debug, args.push_nupkgs_local, args.float
```

Comment:
`black` formatting depends on the trailing comma after last item. If the comma isnt there, it will try to put all the items on the same line. If there is a trailing comma, each item will be on a separate line. So having a trailing comma here would keep the original formatting and the diff would be smaller. Just fyi.

Reply:
Gotcha, thanks.